python_modules/dagster-graphql/dagster_graphql/schema/dagster_types.py

```diff
@@ -1,33 +1,48 @@
 # pylint: disable=missing-graphene-docstring
+from typing import Optional, Union
+
 import graphene
 from dagster_graphql.implementation.events import iterate_metadata_entries
 from dagster_graphql.schema.metadata import GrapheneMetadataEntry
 
 import dagster._check as check
 from dagster.core.snap import PipelineSnapshot
+from dagster.core.snap.dagster_types import DagsterTypeSnap
 from dagster.core.types.dagster_type import DagsterTypeKind
 
-from .config_types import GrapheneConfigType, to_config_type
+from .config_types import GrapheneConfigType, GrapheneConfigTypeUnion, to_config_type
 from .errors import (
     GrapheneDagsterTypeNotFoundError,
     GraphenePipelineNotFoundError,
     GraphenePythonError,
 )
 from .util import non_null_list
 
+GrapheneDagsterTypeUnion = Union[
```

Comment:
Is this being used anywhere?

Reply:
Looks like not, good catch. I did this a while ago so probably part of some unfinished annotation work, but I've removed it for now.